.eslintrc.json

```diff
@@ -1,6 +1,8 @@
 {
     "extends": "airbnb-base",
     "rules": {
-        "no-else-return": 0
+        "no-else-return": 0,
+        "class-methods-use-this": 0,
+        "no-unused-expressions": 0
```

Comment:
class-methods-use-this - I’m not fond of this one because many times you refactor a method between using and not-using ‘this’.

no-unused-expressions - sometimes it’s necessary or preferred.

But I guess we can stick with them, and use the file/line rule exceptions. I’ll remove it.